package-lock.json

```diff
@@ -1538,6 +1538,97 @@
         "@octokit/types": "^2.0.0"
```

Comment:
How did we regenerated this file? I see a lot of changes that looks like accidental. When we run following command agains master:
```sh
$ npm i --save @tim-lai/isomorphic-form-data --verbose
```
and then run diff,
```js
$ git diff
```
I can see only the following changes: 

```diff
diff --git a/package-lock.json b/package-lock.json
index 3a8ce24..eeb5040 100644
--- a/package-lock.json
+++ b/package-lock.json
@@ -1701,6 +1701,14 @@
         "defer-to-connect": "^1.0.1"
       }
     },
+    "@tim-lai/isomorphic-form-data": {
+      "version": "1.0.0",
+      "resolved": "https://registry.npmjs.org/@tim-lai/isomorphic-form-data/-/isomorphic-form-data-1.0.0.tgz",
+      "integrity": "sha512-ukWANS+8w3NCj4Qmnc4zo0XWcoqnDAiqaBXVdh/z6CPtOL+z6I13H3AKOdGzeAaPCeXRL3sPqaPgrSqEgtF8Xg==",
+      "requires": {
+        "formdata-node": "^2.1.1"
+      }
+    },
     "@types/babel__core": {
       "version": "7.1.7",
       "resolved": "https://registry.npmjs.org/@types/babel__core/-/babel__core-7.1.7.tgz",
@@ -6221,6 +6229,26 @@
         "mime-types": "^2.1.11"
       }
     },
+    "formdata-node": {
+      "version": "2.1.1",
+      "resolved": "https://registry.npmjs.org/formdata-node/-/formdata-node-2.1.1.tgz",
+      "integrity": "sha512-XfwCd3Edkt2N81Uh1G72kakbavjEOhRuGdiOV6Ekba/DrFrWFWgkjN4dZn5EPNbjmg+QC+i0ckeYukAbuTRHjw==",
+      "requires": {
+        "@babel/runtime": "7.8.4",
+        "mime-types": "2.1.26",
+        "nanoid": "2.1.11"
+      },
+      "dependencies": {
+        "@babel/runtime": {
+          "version": "7.8.4",
+          "resolved": "https://registry.npmjs.org/@babel/runtime/-/runtime-7.8.4.tgz",
+          "integrity": "sha512-neAp3zt80trRVBI1x0azq6c57aNBqYZH8KhMm3TaB7wEI5Q4A2SHfBHE8w9gOhI/lrqxtEbXZgQIrHP+wvSGwQ==",
+          "requires": {
+            "regenerator-runtime": "^0.13.2"
+          }
+        }
+      }
+    },
     "fragment-cache": {
       "version": "0.2.1",
       "resolved": "https://registry.npmjs.org/fragment-cache/-/fragment-cache-0.2.1.tgz",
@@ -10337,6 +10365,11 @@
       "integrity": "sha512-INOFj37C7k3AfaNTtX8RhsTw7qRy7eLET14cROi9+5HAVbbHuIWUHEauBv5qT4Av2tWasiTY1Jw6puUNqRJXQg==",
       "dev": true
     },
+    "nanoid": {
+      "version": "2.1.11",
+      "resolved": "https://registry.npmjs.org/nanoid/-/nanoid-2.1.11.tgz",
+      "integrity": "sha512-s/snB+WGm6uwi0WjsZdaVcuf3KJXlfGl2LcxgwkEwJF0D/BWzVWAZW/XY4bFaiR7s0Jk3FPvlnepg1H1b1UwlA=="
+    },
     "nanomatch": {
       "version": "1.2.13",
       "resolved": "https://registry.npmjs.org/nanomatch/-/nanomatch-1.2.13.tgz",
diff --git a/package.json b/package.json
index 5bb4d9f..70c8a8a 100644
--- a/package.json
+++ b/package.json
@@ -98,6 +98,7 @@
   "dependencies": {
     "@babel/runtime-corejs2": "^7.0.0",
     "@kyleshockey/object-assign-deep": "^0.4.0",
+    "@tim-lai/isomorphic-form-data": "^1.0.0",
     "btoa": "1.1.2",
     "buffer": "^5.1.0",
     "cookie": "^0.3.1",
```

Reply:
hmm. not seeing what you are seeing. I `npm i` again, and  `package-lock` is current with most recent update.

`form-data` and `isomorphic-form-data` are valid removal. `formdata-node` is valid addition (from `@tim-lai/isomorphic-form-data`). `nanoid` is dependency from `formdata-node`

Reply:
So I guess your `npm i` have updated the dep tree then? Let's research this. Accidental updates of dep tree should not happen unless we explicitly want to.